test/remote/gateways/remote_pay_junction_v2_test.rb

```diff
@@ -5,9 +5,10 @@ def setup
     @gateway = PayJunctionV2Gateway.new(fixtures(:pay_junction_v2))
 
     @amount = 99
-    @credit_card = credit_card('4444333322221111', month: 01, year: 2020, verification_value: 999)
+    @credit_card = credit_card('4444333322221111', month: 01, year: 2021, verification_value: 999)
     @options = {
-      order_id: generate_unique_id
+      order_id: generate_unique_id,
+      billing_address: address()
```

Comment:
It's a wee more Rubyish if you want to leave off the `()` for this method call. I think Python requires `()` to be there for a function call, yeah?

Reply:
Yes, Python require's that, so this is definitely habit. But I did see this being used elsewhere in AM.

Reply:
Ah, yeah, I see it all over... welp shucks.